lib/l10n/app_de.arb

```diff
@@ -71,7 +88,7 @@
         }
     },
     "s_character_count": "Anzahl Zeichen",
-    "s_learn_more": "Mehr\u00a0erfahren",
+    "s_learn_more": "Mehr erfahren",
```

Comment:
Note :point_up:

This is caused by the `ensure_ascii=False` argument in https://github.com/Yubico/yubioath-flutter/blob/2464733f71e5c6c8f188eb4a7eee0d46476be2d7/arb_reformatter.py#L58

in case we prefer keeping these the way they were.

Reply:
good point, I will update and rather use the original `\u` format

Reply:
I don't care either way, I just wanted to point this out so we're aware :smile:

Reply:
Using `ensure_ascii=True` argument will convert all glyphs with code>128 to its `\u...` version which will make the files very hard to read. For example in German localization, instead of `"Ungültige Länge"` we will get `"Ung\u00fcltige L\u00e4nge"`, Japanese will be totally unreadable (`"パスキーが削除されました"` -> `"\u30d1\u30b9\u30ad\u30fc\u304c\u524a\u9664\u3055\u308c\u307e\u3057\u305f"`)

We need to use `ensure_ascii=False`

Reply:
Can we `ensure_ascii=False` and then manually replace certain characters with escape sequences? Eg. `target_text.replace(" ", r"\u00a0")`.

Reply:
We could have a list of replacements for the sequences we use. I think currently it is the No-Break Space (U+00A0) and the Horizontal Ellipsis (U+2026).

I will update the script as part of this PR.